src/extension/background-script/router.ts

```diff
@@ -59,6 +50,29 @@ const routes = {
   lnurl: lnurl,
   lnurlAuth: auth,
   getCurrencyRate: cache.getCurrencyRate,
+  nostr: {
+    generatePrivateKey: nostr.generatePrivateKey,
+    getPrivateKey: nostr.getPrivateKey,
+    setPrivateKey: nostr.setPrivateKey,
+  },
+
+  // Public calls are accessible from inpage scripts
+  public: {
```

Comment:
as we have this new nesting here... and just to review things... can somebody review the router function that resolves those routes?